samples/MiniZinc/data.dzn

```diff
@@ -0,0 +1 @@
+n=3;
```

Comment:
We prefer to use files from real applications. Sample files are used for training the Bayesian classifier and for tests, so it's better if they reflect "real life usage" (with the occasional inconsistencies, etc.). We don't always copy the licensing terms with the file (not sure that's okay), unless there are part of the file's header.